index.html

```diff
@@ -49,5 +52,170 @@ <h3>I am a Software Developer</h3>
 		<img src="images/Illustration.png" alt="shapes background image" class="png">
 	</section>
 
+	<section class="works">
+		<div class="container">
+			<h1>My Recent Works</h1>
+			<div class="project">
+				<img src="images/image 1.png" alt="A yoga lady">
+				<h2>Multi-Post Stories</h2>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy 
+				   text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+				   <ul class="lang">
+					<li><a href="">css</a></li>
+					<li><a href="">html</a></li>
+					<li><a href="">bootstrap</a></li>
+					<li><a href="">Ruby</a></li>
+				</ul>
+			</div>
+			<button class="btn"><a href="">See Project</a></button>
+		</div>
+
+		<div class="container">
+			<div class="project2">
+				<h2>Profesional Art Printing Data</h2>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+				   <ul class="lang">
+					<li><a href="">html</a></li>
+					<li><a href="">bootstrap</a></li>
+					<li><a href="">Ruby</a></li>
+				</ul>
+				<button class="btn2"><a href="">See Project</a></button>
+			</div>
+		</div>
+
+		<div class="container">
+			<div class="project2">
+				<h2>Profesional Art Printing Data</h2>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+				   <ul class="lang">
+					<li><a href="">html</a></li>
+					<li><a href="">bootstrap</a></li>
+					<li><a href="">Ruby</a></li>
+				</ul>
+				<button class="btn2"><a href="">See Project</a></button>
```

Comment:
- [ ] Please add all button effects as mentioned in Figma Design check below:
![Screenshot from 2023-02-19 03-31-14](https://user-images.githubusercontent.com/45226848/219902278-f93fe916-63a8-4465-8f85-0062d6b159d3.png)